src/imagewidget/widget.ts

```diff
@@ -58,12 +58,7 @@ class ImageWidget extends Widget {
     this._scale = value;
     let scaleNode = this.node.querySelector('div') as HTMLElement;
     let transform: string;
-    let percentage = ((value - 1) / 2) * 100 / value;
-    if (value > 1) {
-      transform = `scale(${value}) translate(${percentage}%, ${percentage}%)`;
-    } else {
-      transform = `scale(${value}) translateY(${percentage}%)`;
-    }
+    transform = `scale(${value})`;
```

Comment:
Since this is just a very compact now, may as well put the declaration on a single line:

```typescript
let transform = `scale(${value})`;
```